cuda_core/tests/conftest.py

```diff
@@ -1,19 +1,37 @@
-# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+import multiprocessing
+import os
+
 import helpers
+import pytest
 
 try:
     from cuda.bindings import driver
 except ImportError:
     from cuda import cuda as driver
-import multiprocessing
-
-import pytest
 from cuda.core.experimental import Device, _device
 from cuda.core.experimental._utils.cuda_utils import handle_return
 
 
+def _detect_wsl() -> bool:
```

Comment:
We have two of these floating around already:

```
$ git grep 'def platform_is_wsl'
cuda_bindings/tests/test_cufile.py:def platform_is_wsl():
cuda_core/tests/test_event.py:def platform_is_wsl():
```

It'd be ideal to consolidate them. Not sure how best.